doc/setup/install.rst

```diff
@@ -200,9 +200,9 @@ If you need this feature, you can install the python module like this:
 Fedora
 ''''''
 
-Odoo 11.0 'rpm' package supports Fedora 26.
+Odoo 12.0 'rpm' package supports Fedora 26.
```

Comment:
@d-fence same question, fedora 26 is from nov 2017, Fedora 29 was released just now I think.

Reply:
Actually works with Fedora 26